File: A's/cpp/469A.cpp
#include <bits/stdc++.h>

using namespace std;

int main() {
    int n; cin >> n;

    int x; cin >> x;
    vector<int> X(x);
    for (int i = 0; i < x; i++) {
        cin >> X[i];
    }
    int y; cin >> y;
    vector<int> Y(y);
    for (int i = 0; i < y; i++) {
        cin >> Y[i];
    }

    bool can = true;
    for (int i = 1; i <= n; i++) {
        if (!(find(X.begin(), X.end(), i) != X.end() || find(Y.begin(), Y.end(), i) != Y.end())) {
            can = false;
            break;   
        }
    }
    cout << (can ? "I become the guy." : "Oh, my keyboard!") << endl;

    return 0;
}

File: A's/py/469A.py
def main() -> None:
    n: int = int(input())

    X: list[int] = list(map(int, input().split()))
    x: int = X[0]
    X.pop(0)

    Y: list[int] = list(map(int, input().split()))
    y: int = Y[0]
    Y.pop(0)

    can: bool = True
    for i in range(n + 1):
        if not ((i not in X) or (i not in Y)):
            can = False
            break

    print("I become the guy." if can else "Oh, my keyboard!")

if __name__ == "__main__":
    main()